db/migrations/20200310105940_create_median_kiss_single.sql

```diff
@@ -0,0 +1,26 @@
+-- +goose Up
+CREATE TABLE maker.median_kiss_single
```

Comment:
Sorry to be that guy - stopping in to block merging until timestamps are updated 😬